minutes/2019-12-17.md

```diff
@@ -0,0 +1,65 @@
+# Jakarta EE Platform Call
+
+Date: 2019-12-17
+
+Present:
+
+* Kevin Sutter (IBM)
+* Bill Shannon, Ed Bratt, Dmitry Kornilov (Oracle)
+* Andy McCright (IBM)
+* Cesar Hernandez (Tomitribe)
+* Emily Jiang (IBM)
+* David Blevins (Tomitribe)
+* Scott Stark (Red Hat)
+* Ivar Grimstad (Eclipse Foundation)
+* Tanja Obradovic (Eclipse Foundation)
+* Jonathan Gallimore (Tomitribe)
+* Paul Nicolucci (IBM)
+* Steve Millidge (Payara)
+* Martin Stefanko (Red Hat)
+* Brian Stansberry (Red Hat)
+* Dmitry Kornilov (Oracle)
+* Jesse McConnell (Webtide)
+* Edwin Amoakwa
+* Nathan Erwin
+
+## Agenda
+
+### [KWS] Platform Call frequency in 2020?  Start weekly (Jan 7), except for weeks with PMC call?  And, revisit in Jan or Feb timeframe?
+
+**DECISION:** Continue weekly at least through January. Revisit in February
+
+Practical: Use the same Google Doc for the agenda and add it to the Calendar invite. 
+Ivar will update the calendar.
+
+
+### [SM, KWS] Release Plan
+
+Discussed the release plan:
+
+https://docs.google.com/document/d/18cwfYhEqv_msIdH1DSVDXJc-QaUJ6zjPgmG6JHfLRrA/edit
+
+Discussed:  
+
+* Need an Exception clause at the end of the Namespace Change paragraph.
```

Comment:
Could we somehow indicate that these four items were completed?  We said they were "discussed", but the release plan and guesstimate spreadsheet were also updated accordingly.  Thanks!